### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by codemile**
Describe the need for latest macOS so you can run the app.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -21,6 +21,7 @@ If you have any bug reports, questions, ideas, or unexpected behavior feel free
 - [volta](https://volta.sh/) or Node.js >=20
 - [rust](https://rustup.rs/)
 - [yarn](https://yarnpkg.com/getting-started/install)
+- [macOS](https://www.apple.com/ca/macos/sonoma/) (Mac only) macOS Sonoma (14.2 or higher)
```
Comment (abrenneke) [inline CONTRIBUTING.md:24]: Sonoma definitely isn't required, I'm on 13.6. I think it's 12.3 minimum or something.

Comment (codemile) [reply CONTRIBUTING.md]: fixed.

**Review by abrenneke — APPROVED** [review]
Thanks!

**Comment by abrenneke** [comment]
@all-contributors please add @codemile for docs

**Pull request merged**